client/src/components/Form/components/formSections/MulighetForArbeidSection/AvventendePeriode.tsx

```diff
@@ -39,6 +40,11 @@ const AvventendePeriode = ({ updateMfa, mfaPeriode, errors, index }: AvventendeP
                     updateMfa(updatedSchema);
                 }}
             />
+            {mfaPeriode.avventendePeriode != undefined &&
+                <div style={{ marginTop: '-0.5rem', marginBottom: '2rem' }}>
+                    Periode valgt: {moment(mfaPeriode.avventendePeriode[0]).format("Do MMMM YYYY")} - {moment(mfaPeriode.avventendePeriode[1]).format("Do MMMM YYYY")}
+                </div>
+            }
```

Comment:
Samme gjelder her, som for kommentaren over: flytte dette inn i komponenten `RangePicker`